### PR TITLE
[jsk_tools] Merge 99.jsk_tools.[bash|zsh] to 99.jsk_tools.sh

### PR DESCRIPTION
**Pull request by wkentaro**
@k-okada @garaemon I merged them.

issue: https://github.com/jsk-ros-pkg/jsk_common/issues/869

### CODE REVIEW

**Comment by garaemon** [comment]
travis not yet started.. wait a little bit more

File: jsk_tools/env-hooks/99.jsk_tools.bash [hunk 1]
```diff
@@ -1,117 +1,6 @@
...
-    else
-        node=$(rostopic info $1 | percol | sed -e 's%.* \* \(/[/a-zA-Z0-9_]*\) .*%\1%')
-    fi
-    if [ "$node" != "" ] ; then
-        rosn $node
-    fi
-}
-
+# source 99.jsk_tools.sh from same directory as this file
+_THIS_DIR=$(buildin cd -q "`dirname "${BASH_SOURCE[0]}"`" > /dev/null && pwd)
```
Comment (k-okada) [inline jsk_tools/env-hooks/99.jsk_tools.bash]: bash does not have neither buildin nor cd -q
```
k-okada@kokada-t440s:~/ros/indigo/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks$ source 99.jsk_tools.bash 
buildin: command not found
bash: /99.jsk_tools.sh: No such file or directory
k-okada@kokada-t440s:~/ros/indigo/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks$ emacs -nw  99.jsk_tools.bash 

[3]+  Stopped                 emacs -nw 99.jsk_tools.bash
k-okada@kokada-t440s:~/ros/indigo/src/jsk-ros-pkg/jsk_common/jsk_tools/env-hooks$ source 99.jsk_tools.bash 
bash: cd: -q: invalid option
cd: usage: cd [-L|[-P [-e]] [-@]] [dir]
bash: /99.jsk_tools.sh: No such file or directory
```

Comment (k-okada) [inline jsk_tools/env-hooks/99.jsk_tools.bash]: fixed in 42f0892

**Comment by wkentaro** [comment]
sorry, it should be `builtin cd` without `-q`.
thanks for fixing it.

**Comment by wkentaro** [comment]
I send PR: https://github.com/jsk-ros-pkg/jsk_common/pull/874/files

**Pull request merged**